frontends/web/src/api/account.ts

```diff
@@ -296,13 +296,29 @@ export const getReceiveAddressList = (code: AccountCode) => {
   };
 };
 
+// TextMemo represents a slip-0024 text memo.
+export type TTextMemo = {
+  note: string;
+};
+
+// PaymentRequest contains the data needed to fulfill a slip-0024 payment request.
+// Text memos are the only memo type supported, currently.
+export type TPaymentRequest = {
+  recipientName: string;
+  memos: TTextMemo[];
+  nonce: string | null;
+  totalAmount: number;
+  signature: string;
+};
+
```

Comment:
Adding this and converting from the [slip24 types](https://github.com/pocketbitcoin/request-address/blob/main/src/slip24.ts#L3) seems like overkill, as they are virtually the same. Why not send the slip24 object directly to the backend and use their format to deserialize?